README.md

```diff
@@ -863,6 +863,19 @@ var signIn = new OktaSignIn(config);
     }
     ```
 
+### Colors
+
+You can control the main style of the widget by setting the following config options. The colors need to be in `6-digit hex` format.
+
+- **colors.brand:** Set the brand color of the widget. This will affect the color of all the primary buttons of the widget.
```

Comment:
Suggestion:

> Sets the brand (primary) color. Colors must be in hex format, like `#008000`.

Reply:
(For now, I want to stay away from listing everything that colors.brand will affect)